src/cascadia/LocalTests_TerminalApp/TerminalApp.LocalTests.AppxManifest.prototype.xml

```diff
@@ -3,49 +3,48 @@
 
   <!-- This file is used as the Appxmanifest for tests that _need_ to run in a
   packaged environment. It will be copied to the test's OutDir as part of the
-  PostBuid step. It's highly similar to the "PackagedCwaFullTrust" manifest that
+  PostBuid step. It's highly similar to the "SUA" manifest that
   TAEF ships with, with the following modifications:
 
   1. All of our winrt types are included in this manifest, including types from
-     MUX.dll. Should this list of types ever change, we'll need to manually
-     update this file. The easiest way of doing that is deploying the app from
-     VS, then copying the Extensions from the Appxmanifest.xml that's generated
-     under `src/cascadia/CascadiaPackage/bin/$(platform)/$(configuration)/appx`.
+     MUX.dll. This is done in a custom build step that automatically adds each
+     of our types to this manifest.
 
   2. We also _NEED_ the two vclibs listed under the `PackageDependency` block.
 
-  If your test fails for whatever reason, it's likely possible you're testing a
-  type that's _not_ included in this file for some reason. So, here be dragons. -->
+  -->
 
   <Identity Name="TerminalApp.LocalTests.Package"

```

Comment:
Does this work in the gates now? Do we still need to call it LocalTests?

Reply:
I honestly don't know yet. I was going to do that as a part of a follow-up PR. I wanted to just get the tests unblocked for now. I think as part of the whole "make the tab tests work" bit, I'll have a chance to loop back on that subject.